packages/wrangler/src/logger.ts

```diff
@@ -36,6 +37,7 @@ class Logger {
 
 	loggerLevel: LoggerLevel = (getLogLevelFromEnv() as LoggerLevel) ?? "log";
 	columns = process.stdout.columns;
+	debugLogs: string[] = [];
```

Comment:
Are concerned that in a long running Wrangler session, like a tail or a dev command, this might result in a large amount of memory usage?
I wonder if we should make this a sliding window with a fixed size?

Reply:
Have now capped this to 10K rows.

Reply:
I agree with @mrbbot—especially if this is to catch hard to debug issues, it'd be nice to catch things like Node OOM errors that would just crash the process without running the `writeFileSync`

Reply:
Would you want to clean up that file if the process ends successfully? Or truncate it on the next `wrangler` start? Or just create a new file each time?